2020-an-environment-for-sustainable-research-software-in-germany-and-beyond/manuscript.md

```diff
@@ -0,0 +1,2020 @@
+> *hal9k* and publish it through the university library’s software
+> portal, where they get a DOI (Digital Object Identifier) for the
+> version as well as a concept DOI for any future versions of the
+> library. Working with the RSE, Kim has gained a good understanding of
+> some methods in software engineering, and she’s thrilled because this
+> also means she’ll be able to get a job with a local tech company once
+> her fixed-term contract has run out.  
+> Kim passes her PhD - of which *hal9k* is an important part - with
+> flying colours, and soon citations to her library start appearing in
+> the researchonomic literature. To Kim’s surprise, she also reads a
+> blog post about a citizen science maker project which has used *hal9k*
+> to process researchonomic data measured in a neighbourhood of her
+> hometown. She is invited to give a talk at the local office of Siren,
+> a global tech company, which look to adopt *hal9k*, and pay Kim
+> handsomely. So handsomely in fact, that Kim can pay a student
+> assistant for a full year from the money.
+
+Our credibility as researchers in society hinges on the notion of proper
+research conduct, also known as “good research practice”. The
+digitalization of research has introduced complex digital research
+outputs, such as software and data sets. Although first
+recommendations (Katerbow and Feulner 2018) and policies (Scheliga et
+al. 2019) exist, they are far from being widely adapted. It is still
+somewhat unclear how to translate good research practice into research
+software practice, for example in terms of validity and reproducibility,
+but also pertaining to the responsible use of resources.
+
+While it is obvious that software qualifies as a potentially re-usable
+digital artifact, the additional benefit of not just reproducing a given
+scenario, but transferring software use to new problems, domains, and/or
+applications, justifies developing research software with a long-term
+perspective as *sustainable research software*.
+
+In order to support research, a sustainable software must be
+correct (Hatton 2007; Chang et al. 2006; Matthews 2007), validatable,
+understandable, documented, publicly released, adequately published
+(i.e. in persistently identifiable form as software source code (Smith
+et al. 2016), and potentially in an additional paper which describes the
+software concept, design decisions and development rationale), actively
+maintained, and  (Merali 2010; Barnes 2010; Tse 2010).
+
+To make it possible to reproduce computationally supported research, a
+software that has been used must continue to exist in the first place,
+but also continue to be usable, understandable, and reproducible in the
+evolving software and hardware environment. Moreover, the software
+should support reuse scenarios to avoid duplication of efforts, and
+unneeded drain of resources. If it is publicly funded, it should also be
+open source and freely accessible.
```

Comment:
```suggestion
To make sofware based research (and with that almost any research)
reproducible, the software used must continue to exist. Further it must
continue to be usable, understandable, and results must be return the
same results (or document potential changes and bug fixes) in the
evolving software and hardware environment. Moreover, the software
should support reuse scenarios to avoid duplication of efforts, and
unneeded drain of resources. If it is publicly funded, software should be
open source and freely accessible.
```

I tried to make it a little less clunky. Hope it worked :) 
Also I tried to avoid "to make it reproducible, software has to be reproducible".

Reply:
A small remark: "and results must be return the same results" should be sometime like "and return the same results"